org.lflang/src/org/lflang/TargetProperty.java

```diff
@@ -58,7 +58,14 @@
  * @author{Marten Lohstroh <marten@berkeley.edu>}
  */
 public enum TargetProperty {
-
+    /**
+     * Directive to allow including OpenSSL libraries and process HMAC authentication.
+     * FIXME: The option will be updated from boolean to string for support of various options.
```

Comment:
Let's not leave this `FIXME` here. I think we can just remove it.

Reply:
Fixed it.